tests/test_encodings.py

```diff
@@ -488,6 +488,19 @@ def test_mds_scalar(self, encoding: str, decoded: Union[int, float], encoded: by
         dec = mdsEnc.mds_decode(encoding, encoded)
         assert dec == decoded
 
+    def test_varints(self):
+        from streaming.base.format.mds.encodings import mds_decode, mds_encode
+        for x in range(-700, 700, 7):
+            y = mds_encode('varint', x)
+            z = mds_decode('varint', y)
+            print(x, y, z)
```

Comment:
do we need the print statements both here and below?

Reply:
Also, can we test ints and their expected lengths that we would encode through tokenization? For example, the `vocab_size` of many models customers train is between 50 and 150k. Testing ints in that range would be useful

Reply:
your spurious print statement life debt is paid